plugins/survey/src/index.ts

```diff
@@ -14,90 +14,73 @@
 //
 
 import type { IntlString, Plugin } from '@hcengineering/platform'
-import type { AttachedDoc, Class, CollectionSize, Markup, Ref } from '@hcengineering/core'
-import type { Attachment } from '@hcengineering/attachment'
+import type { Class, Mixin, Ref, Type } from '@hcengineering/core'
 import { Asset, plugin } from '@hcengineering/platform'
+import {
+  CheckboxesOption,
+  Checkboxes,
+  Info,
+  Question,
+  QuestionDataEditor,
+  RadioButtonsOption,
+  RadioButtons,
+  Rank,
+  Survey,
+  QuestionData
+} from './types'
 
-/**
- * @public
- */
-export type Rank = string
-
-/**
- * @public
- *
- * ∈ [0.0, 100.0]
- */
-export type Percentage = number
-
-/**
- * @public
- *
- * Arbitrary survey item
- */
-export interface SurveyItem extends AttachedDoc {
-  rank: Rank
-}
-
-/**
- * @public
- *
- * Interactive survey item, e.g. a question
- */
-export interface SurveyInteractiveItem extends SurveyItem {
-  title: string
-}
+export * from './types'
 
 /**
  * @public
- *
```

Comment:
BTW, what are these `@public` JS-doc sections for? Do we really need'em? Not a big deal, just curious